### PR TITLE
[PIE-2154] Change passive voice to active voice

### PR DESCRIPTION
**Pull request by sgregglives**
Signed-off-by: Steven Gregg <steven.gregg@consensys.net>

## Pull Request Description
Change passive voice to active where appropriate
## Fixed Issue(s)

PIE-2154

### CODE REVIEW

File: docs/HowTo/Extending-EthSigner/Add-Signer.md [hunk 1]
```diff
@@ -4,26 +4,26 @@ description: Adding an external signer to EthSigner
 
 # Creating an external signer for EthSigner
 
-EthSigner supports the implementation of additional external signers in same way as Hashicorp Vault.
+EthSigner supports the implementation of additional external signers in the same way as Hashicorp Vault.
```
Comment (rain-on) [inline docs/HowTo/Extending-EthSigner/Add-Signer.md]: The Ethsigner source code is structured to allow additional signing mechanisms to be added by following an established pattern:

File: docs/HowTo/Extending-EthSigner/Add-Signer.md [hunk 2]
```diff
@@ -4,26 +4,26 @@ description: Adding an external signer to EthSigner
...
-1. To ensure the module for the new signer is built, add the module to the `settings.gradle` file.
+1. To include the module in the build, add the module to the `settings.gradle` file.
 
 1. In the new module, create a concrete child class of `SignerSubCommand`.
 
     The new `SignerSubCommand` must include the parameters (tagged with PicoCLI annotations) to
     initialize your signer.
 
-1. Update the `createSigner()` function to be the entry point to create and return your signer.
+1. Make the `createSigner()` function the entry point to create and return your signer.
```
Comment (rain-on) [inline docs/HowTo/Extending-EthSigner/Add-Signer.md]: Make the `createSignerFactory()` function  the entry point to create and return your TransactionSignerProvider (a class capable of supplying 1 or more TransactionSigners (which in turn is able to sign transactions using the mechanism in development)).

Comment (rain-on) [reply docs/HowTo/Extending-EthSigner/Add-Signer.md]: Your newly created TransactionSupplierProvider should be exposed only as such (i.e. as the base interface).

Comment (sgregglives) [reply docs/HowTo/Extending-EthSigner/Add-Signer.md]: I'm unsure about using "should be" here. Are we using it to express advice or an expectation?

File: docs/HowTo/Extending-EthSigner/Add-Signer.md (hunk 1 above)
Comment (sgregglives) [reply docs/HowTo/Extending-EthSigner/Add-Signer.md]: By pattern do you mean the procedure that follows the statement to be changed?

Comment (MadelineMurray) [reply docs/HowTo/Extending-EthSigner/Add-Signer.md]: Fixed

File: docs/HowTo/Extending-EthSigner/Add-Signer.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Extending-EthSigner/Add-Signer.md]: Reverted to original - comments covered by following sentence and note.

**Pull request closed without merging**